### PR TITLE
feature/v1-030

### PR DESCRIPTION
**Pull request by Julia-Maliutsina**
Overview:
1. Shared components Image, Course and Course info, that can be reused on other pages.
2. Styled page with user's courses list.

### CODE REVIEW

File: frontend/src/pages/MyCourses/Course.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+interface Props {
+  title: string;
+  description: string;
+  language: string;
+  duration: string;
+  lessons: string;
+  link: string;
+}
+
+const CourseItem: React.FC<Props> = ({ title, description, language, duration, lessons, link }) => (
```
Comment (k-krasilnikova) [inline frontend/src/pages/MyCourses/Course.tsx:25]: Looks like common component, but check it please with Irina

File: frontend/src/pages/MyCourses/MyCourses.tsx [hunk 2]
```diff
@@ -2,9 +2,40 @@ import React from 'react';
 
 import { AuthorizedLayout } from 'components/Layout';
 
+import CourseItem from './Course';
+import { PageContainer } from './styled';
+
+const INITIAL = {
```
Comment (k-krasilnikova) [inline frontend/src/pages/MyCourses/MyCourses.tsx:8]: Define it like `const INITIAL_COURSES = []`
And map over this array on ui

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: frontend/src/pages/MyCourses/Course.tsx (hunk 1 above)
Comment (Julia-Maliutsina) [reply frontend/src/pages/MyCourses/Course.tsx:25]: It is almost the same on all pages. The only difference is its size and buttons. Should I create common component for all pages that receives size and buttons?

File: frontend/src/components/Image/Image.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  imageUrl?: string;
+}
+const SIZES = {
+  small: 'small',
+  large: 'large',
+};
+
+const Image: React.FC<Props> = ({ imageUrl, size }) => (
+  <ImageContainer
+    style={size === SIZES.small ? { width: 150, height: 50 } : { width: 300, height: 200 }}
```
[the remark this answers is not in the document]
Comment (Julia-Maliutsina) [reply frontend/src/components/Image/Image.tsx:18]: I have some problems with passing props to styled.tsx

**Review by hauteballe — APPROVED** [review]

File: frontend/src/components/Avatar/styled.ts [hunk 4]
```diff
@@ -1,7 +1,21 @@
...
   display: 'inline-block',
   margin: '1% auto',
-});
+  width: '100px',
+  height: '100px',
+  ...(size === 'large' && {
+    width: '100px',
+    height: '100px',
+  }),
+  ...(size === 'small' && {
```
Comment (k-krasilnikova) [inline frontend/src/components/Avatar/styled.ts:17]: From constants!!
Try to minimize using of different strings in code

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

**Review by hauteballe — APPROVED** [review]

**Review by k-krasilnikova — APPROVED** [review]

**Pull request merged**